OptimizelySDKCore/OptimizelySDKCore/Optimizely.m

```diff
@@ -300,7 +300,7 @@ - (NSString *)getFeatureVariableValueForType:(NSString *)variableType
         OPTLYVariation *variation = decision.variation;
         OPTLYVariableUsage *featureVariableUsage = [variation getVariableUsageForVariableId:featureVariable.variableId];
         
-        if (featureVariableUsage) {
+        if (featureVariableUsage && variation.featureEnabled) {
```

Comment:
Log message needs to be fixed as pointed out in other SDKs here as well. The new scenario is a situation where user is in variation, but has feature disabled and hence the default value.

Reply:
Suggested changes made.